viewer_clients/cmdline_client/cmd_line_client.py

```diff
@@ -240,12 +243,23 @@ def add_filter_conditions(report_filter, filter_str):
     severity, checker, path = map(lambda x: x.strip(), filter_str.split(':'))
 
     if severity:
-        report_filter.severity = shared.ttypes.Severity._NAMES_TO_VALUES[severity.upper()]
+        report_filter.severity\
+            = shared.ttypes.Severity._NAMES_TO_VALUES[severity.upper()]
     if checker:
         report_filter.checkerId = '*' + checker + '*'
     if path:
         report_filter.filepath = path
 
+
+def add_server_arguments(parser):
+    parser.add_argument('--host', type=str, dest="host",
+                        default='localhost',
+                        help='Server host.')
+    parser.add_argument('-p', '--port', type=str, dest="port",
+                        default=11444,
```

Comment:
Though I know this is just a DRY extraction, the default port number `11444` is no longer in line with the default view-port number `8001` set here: https://github.com/Ericsson/codechecker/blob/master/codechecker/CodeChecker.py#L416